### PR TITLE
Minor code quality changes

### PR DESCRIPTION
**Pull request by cjbassi**

### CODE REVIEW

File: i3wsgroups/i3_workspace_groups.py [hunk 1]
```diff
@@ -146,7 +144,7 @@ class WorkspaceGroupsError(Exception):
     pass
 
 
-class ActiveGroupContext(object):
+class ActiveGroupContext():
```
Comment (infokiller) [inline i3wsgroups/i3_workspace_groups.py]:
```suggestion
class ActiveGroupContext:
```

File: i3wsgroups/i3_workspace_groups.py [hunk 2]
```diff
@@ -164,7 +162,7 @@ def get_workspace(self, tree):
         return active_group_workspaces[0]
 
 
-class FocusedGroupContext(object):
+class FocusedGroupContext():
```
Comment (infokiller) [inline i3wsgroups/i3_workspace_groups.py]:
```suggestion
class FocusedGroupContext:
```

File: i3wsgroups/i3_workspace_groups.py [hunk 3]
```diff
@@ -174,7 +172,7 @@ def get_workspace(self, tree):
         return tree.find_focused().workspace()
 
 
-class NamedGroupContext(object):
+class NamedGroupContext():
```
Comment (infokiller) [inline i3wsgroups/i3_workspace_groups.py]:
```suggestion
class NamedGroupContext:
```

File: i3wsgroups/i3_workspace_groups.py [hunk 4]
```diff
@@ -193,7 +191,7 @@ def get_workspace(self, tree):
         return group_to_workspace[self.group_name][0]
 
 
-class WorkspaceGroupsController(object):
+class WorkspaceGroupsController():
```
Comment (infokiller) [inline i3wsgroups/i3_workspace_groups.py]:
```suggestion
class WorkspaceGroupsController:
```

**Comment by infokiller** [comment]
Thanks! added suggestions for removing the parentheses from the class definitions as well.

**Comment by cjbassi** [comment]
Fixed!

**Pull request merged**